src/app/beer_garden/api/http/handlers/vbeta/file.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+import io
+
+from brewtils.resolvers.parameter import BYTES_PREFIX
+
+from beer_garden.api.http.authorization import Permissions, authenticated
+from beer_garden.api.http.base_handler import BaseHandler
+from beer_garden.db.mongo.models import RawFile
+
+
+class RawFileAPI(BaseHandler):
+    @authenticated(permissions=[Permissions.READ])
+    async def get(self):
+        """
+        ---
+        summary: Retrieve a File
+        parameters:
+          - name: file_id
```

Comment:
I'm assuming this will become part of the URL `/api/vbeta/file/<id>`

Reply:
👍🏼